### ISSUE TITLE
config file

### ISSUE THREAD

**Issue by SdidevelopWork** [Fmstrat/winapps#36]
You need to create a ~/.config/winapps/winapps.conf configuration. Exiting...

While I was creating the file ‍‍‍‍~/.config/winapps/winapps.conf

**Comment by Tiebe** [comment]
Did you run the script as root? It was not made for this, because then it will look in /home/root/.config/winapps.conf

**Comment by Fmstrat** [comment]
Yea, I'm not sure what's being asked here unfortunately.

**Comment by liaoyaonline** [comment]
我也遇到了这个问题。。。。。。后面解决了，但我不知道为什么解决了。说一下我的建议。使用sudo 执行，而不是在root下执行，假如使用KVM虚拟机的话，记得winapps.conf里面除了前面两行，其他的不需要取消注释。。。。。

**Comment by Citarosoftwarerepairs** [comment]
Yeah you shouldn't run as sudo (which is what I did and it'll say that)
